### PR TITLE
[16.0][IMP] stock_release_channel_shipment_advice_deliver: Add option auto_delivery and allow to print in-process shipment advice

### PR DESCRIPTION
**Pull request by tuantrantg**

### CODE REVIEW

File: stock_release_channel_shipment_advice_deliver/models/stock_release_channel.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ class StockReleaseChannel(models.Model):
     in_process_shipment_advice_ids = fields.One2many(
         "shipment.advice", compute="_compute_in_process_shipment_advice_ids"
     )
+    auto_deliver = fields.Boolean()
```
Comment (jbaudoux) [inline stock_release_channel_shipment_advice_deliver/models/stock_release_channel.py:43]: Can you add as a condition in `_compute_is_action_deliver_allowed` ?

File: stock_release_channel_shipment_advice_deliver/models/stock_release_channel.py [hunk 2]
```diff
@@ -63,6 +64,7 @@ def _compute_is_action_deliver_allowed(self):
                 )
                 and bool(rec.picking_to_plan_ids)
                 and rec.shipment_planning_method != "none"
+                and rec.auto_deliver
```
Comment (tuantrantg) [inline stock_release_channel_shipment_advice_deliver/models/stock_release_channel.py:67]: hi @jbaudoux `auto_deliver` is added in `_compute_is_action_deliver_allowed`, please review it

**Comment by tuantrantg** [comment]
hi @jbaudoux could you please review it again

**Review by jbaudoux — APPROVED** [review]

**Comment by jbaudoux** [comment]
Merging as failing test does not seem related to this

**Pull request merged**